virtual_rainforest/models/animals/model.py

```diff
@@ -0,0 +1,118 @@
+"""The :mod:`~virtual_rainforest.animal.model` module creates a
```

Comment:
A lot of the links in this doc string are broken, i.e. they are `~virtual_rainforest.animals` when they should be `~virtual_rainforest.models.animals`

Reply:
Right! Thanks